### ISSUE TITLE
Errors When Installing SQLiteHook

### ISSUE THREAD

**Issue by aaronksaunders** [capacitor-community/sqlite#246]
**Describe the bug**
Error when installing sqlite hook

**To Reproduce**

`npm i --save-dev react-sqlite-hook@latest`

**Expected behavior**
no errors

```
aaronksaunders@Aarons1ProM1Pro mobile % npm i --save-dev react-sqlite-hook@latest
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: cnrm-app@0.0.1
npm ERR! Found: @capacitor-community/sqlite@3.4.2-2
npm ERR! node_modules/@capacitor-community/sqlite
npm ERR!   dev @capacitor-community/sqlite@"^3.4.2-2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor-community/sqlite@"^3.4.1-1" from react-sqlite-hook@2.1.6
npm ERR! node_modules/react-sqlite-hook
npm ERR!   dev react-sqlite-hook@"2.1.6" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/aaronksaunders/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aaronksaunders/.npm/_logs/2022-03-28T19_51_00_103Z-debug-0.log
aaronksaunders@Aarons1ProM1Pro mobile % 
```

**Comment by jepiqueau** [comment]
@aaronksaunders fix in `react-sqlite-hook`release 2.1.7

**Comment by jepiqueau** [comment]
@aaronksaunders feel free to close the issue when tested
